.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
![image](https://github.com/Peter-Kibet/leaderboard-project/assets/93039359/8cb54e36-985d-4e9f-87d0-3bbf9e8189aa)

- [ ] It would be nice if you make the highlighted link work because it is broken right now 😨 Kindly fix it or remove it to make your readme descriptive and professional 👍